acceptance/tests/resource/mount/modify.rb

```diff
@@ -3,37 +3,44 @@
 confine :except, :platform => ['windows']
 confine :except, :platform => /osx/ # See PUP-4823
 confine :except, :platform => /solaris/
-confine :except, :platform => /aix/
 
-fstab = '/etc/fstab'
+require 'puppet/acceptance/mount_utils'
+extend Puppet::Acceptance::MountUtils
+
 name = "pl#{rand(999999).to_i}"
 new_name = "pl#{rand(999999).to_i}"
 
 agents.each do |agent|
+  fs_file = filesystem_file(agent)
+
   teardown do
-    #(teardown) restore the fstab file
-    on(agent, "mv /tmp/fstab #{fstab}", :acceptable_exit_codes => (0..254))
     #(teardown) umount disk image
     on(agent, "umount /#{name}", :acceptable_exit_codes => (0..254))
     #(teardown) delete disk image
+    if agent['platform'] =~ /aix/
+      on(agent, "yes | rmfs /dev/#{name}")
+      on(agent, "yes | rmlv #{name}")
```

Comment:
@johnduarte yeah, good point. I'll add an `:acceptable_exit_codes` bit like the umount step.